### DISCUSSION TITLE
Update row cell with strict parameter set to false.

### DISCUSSION THREAD

**Discussion by jack-vinitsky** [skywayskase/PSSmartsheet#7] [Q&A] [answered]
Greetings,

Just started using this module and I am a big fan.

I see that directly updating contact columns is not supported as of yet but I was wondering if there is an option to set the strict property to false for a given cell update.  This would allow a little more flexibility in updating certain column types like contact.

**Comment by jack-vinitsky** [comment]
> Greetings,
> 
> Just started using this module and I am a big fan.
> 
> I see that directly updating contact columns is not supported as of yet but I was wondering if there is an option to set the strict property to false for a given cell update. This would allow a little more flexibility in updating certain column types like contact.

Just noticed the **-OverrideValidation** switch on the New-SSCellObject cmdlet.  However it does not seem to work with Contact columns in that I can't just update it the cell with a text value that is not an email address.

**Comment by skywayskase** [comment, marked as answer]
Hi @jack-vinitsky 
Thanks for bringing this to my attention. I've gone ahead and added a boolean parameter called `strict` to `New-SSCellObject`. It defaults to `$true`, but you can set it to `$false` when you use the cmdlet. Also, I've updated `-OverrideValidation` to automatically set `-strict` to `$false` when it's used.

**Reply by jack-vinitsky** [reply]
Thank you so much!
